Apps/Inc/SendTritium.h

```diff
@@ -0,0 +1,28 @@
+#ifndef __SENDTRITIUM_H
+#define __SENDTRITIUM_H
+
+#define MOTOR_MSG_PERIOD 100
+#define FSM_PERIOD 20
+#define DEBOUNCE_PERIOD 2 // in units of FSM_PERIOD
+#define MOTOR_MSG_COUNTER_THRESHOLD (MOTOR_MSG_PERIOD)/(FSM_PERIOD)
+
+#ifdef __TEST_SENDTRITIUM
+// Inputs
+extern bool cruiseEnable;
+extern bool cruiseSet;
+extern bool onePedalEnable;
+extern bool chargeEnable;
```

Comment:
I couldn't find a chargeEnable in ReadCarCAN, but should I change the regenEnable variable/getter function name to chargeEnable so that they're more clearly separated from regen braking?

Reply:
@KnockbackNemo If that's the case, then it's backward. It should be `chargeEnable` in ReadCarCAN (since ReadCarCAN is not technically supposed to know about regen, it's gating all charging). SendTritium should be the one with `regen_enable`.